packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -262,6 +269,119 @@ describe('RestServer (integration)', () => {
     expect(response.get('Access-Control-Allow-Credentials')).to.equal('true');
   });
 
+  it('exposes "GET /openapi.json" with openApiSpec.servers', async () => {
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+        openApiSpec: {
+          servers: [{url: 'http://127.0.0.1:8080'}],
+        },
+      },
+    });
+    const greetSpec = {
+      responses: {
```

Comment:
~~This OpenAPI snippet is repeated in many tests. Please extract it into a helper function, e.g a function that adds a dummy route.~~~

Do we really need a route to verify how top-level spec properties are handled? Isn't in enough to render the spec for an empty app? Such app should still produce `servers` property of the spec. See the test case "exposes endpoints with openApiSpec.endpointMapping" for an example.

Leaving out the route we don't need in the test will make the test code much simpler and easier to understand. Please remove it.